easybuild/easyconfigs/b/BamTools/BamTools-2.4.0-foss-2015b.eb

```diff
@@ -15,17 +15,10 @@ description = """BamTools provides both a programmer's API and an end-user's too
 
 toolchain = {'name': 'foss', 'version': '2015b'}
 
-sources = ['v%(version)s.tar.gz']
 source_urls = ['https://github.com/pezmaster31/bamtools/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['f1fe82b8871719e0fb9ed7be73885f5d0815dd5c7277ee33bd8f67ace961e13e']
 
 builddependencies = [('CMake', '3.4.1')]
 
-files_to_copy = ["bin", "lib", "include", "docs", "LICENSE", "README"]
-
-sanity_check_paths = {
-    'files': ["bin/bamtools", "include/shared/bamtools_global.h", "lib/libbamtools.a",
-              "lib/libbamtools.so", "lib/libbamtools-utils.a", "lib/libjsoncpp.a"],
-    'dirs': ["include/api", "docs"]
-}
```

Comment:
@boegel - is the sanity check removed because that's in the `easyblock` now?

Reply:
@verdurin Yes, exactly (it actually already was, there was no reason at all to have it in the easyconfig file).

Reply:
@boegel - okay, thanks.